packages/angular/src/generators/application/lib/normalized-schema.ts

```diff
@@ -11,6 +11,4 @@ export interface NormalizedSchema extends Schema {
   e2eProjectName: string;
   e2eProjectRoot: string;
   parsedTags: string[];
-  ngCliSchematicAppRoot: string;
-  ngCliSchematicE2ERoot: string;
```

Comment:
These were no longer needed. It's a leftover of invoking the Angular CLI schematic, which we removed some time ago.